server/models/Transaction.js

```diff
@@ -864,7 +864,7 @@ Transaction.createHostFeeShareTransactions = async (
   }
 
   // We use the Host Fee amountInHostCurrency/hostCurrency as a basis
-  const amount = calcFee(hostFeeTransaction.amountInHostCurrency, hostFeeSharePercent); // TODO without taxes
```

Comment:
This was a TODO I initially added when starting the project, but turned out this was not the best place to fix this